### PR TITLE
Add song progress bar

### PR DESCRIPTION
**Pull request by mas8074**
There were a ton of problems when I rebased. I'm not sure why, but I know it's missing some things. Though I can't figure out what since I didn't work on them. Please take a look at the changes coming in and look for anything you think might be missing.

### CODE REVIEW

File: src/mixing/mixing.py [hunk 1]
```diff
@@ -1,26 +1,14 @@
 import queue
```
Comment (cha2251) [inline src/mixing/mixing.py:1]: We need all of this

File: src/output_queue/output_queue.py [hunk 2]
```diff
@@ -1,12 +1,10 @@
 from queue import PriorityQueue, Empty
 import mido
```
Comment (cha2251) [inline src/output_queue/output_queue.py:2]: Keep all this

File: src/user_interface/main_page.py [hunk 3]
```diff
@@ -29,7 +84,7 @@ def __init__(self, shutdown, mixing_system, file_input):
         self.home_page.nav_settings.clicked.connect(self.go_to_settings_page)
         self.home_page.pick_song_lambda = lambda song: self.update_playing_page_song(song)
 
-        self.play_page = PlayingPage(mixing_system=self.mixing_system)
```
Comment (cha2251) [inline src/user_interface/main_page.py:32]: Need this

File: src/user_interface/main_page.py [hunk 4]
```diff
@@ -55,7 +110,7 @@ def go_to_settings_page(self):
 
     def update_playing_page_song(self, song_name):
         self.play_page.set_song(song_name)
-        self.file_input.openFile(song_name)
```
Comment (cha2251) [inline src/user_interface/main_page.py:58]: Need this

File: test/output_queue_tests/output_queue_test.py [hunk 5]
```diff
@@ -6,15 +6,13 @@
```
Comment (cha2251) [inline test/output_queue_tests/output_queue_test.py:6]: Keep all

**Review by cha2251 — CHANGES_REQUESTED** [review]
Theres a lot that got deleted here. I think I caught most of my stuff, but it might be better just to start fresh and try to rebase your changes on main

File: README.md [hunk 6]
```diff
@@ -31,7 +31,7 @@ https://cha2251.github.io/rit-player-piano/
 - Run with `python -m src.common.main` to prevent package errors
 
 # Requirements
-- If running Linux, install dependencies with `sudo apt-get install -y libasound2-dev pkg-config libjack-dev portaudio19-dev`
```
[the remark this answers is not in the document]
Comment (mas8074) [reply README.md:34]: Yeah, okay. I'm going to delete this and see if I can revert my changes to before the rebase. I accepted all of the changes from main, I don't understand why this happened.

**Pull request closed without merging**